be/src/vec/exec/format/json/new_json_reader.cpp

```diff
@@ -330,6 +330,10 @@ Status NewJsonReader::get_parsed_schema(std::vector<std::string>* col_names,
         objectValue = _json_doc;
     }
 
+    if (!objectValue->IsObject()) {
+        return Status::DataQualityError("JSON data is not an object.");
```

Comment:
add first n bytes of data in error message

Reply:
Does this issue only occur in the TVF?

Reply:
Yes , only in select tvf . streamload will return false

Reply:
What do you mean by `streamload will return false`. I'm concerned that this change might affect other load.

Reply:
this function just for scanner to read json file when in tvf , other load will not call  this function 
just with VFileScanner::_get_block_wrapped --> NewJsonReader::get_next_block --> NewJsonReader::_read_json_column --> NewJsonReader::_vhandle_simple_json --> NewJsonReader::_set_column_value

Reply:
and I think I could print origin type instead of n bytes...